src/table/selection/selection-control.tsx

```diff
@@ -100,10 +100,6 @@ export function SelectionControl({
       >
         {selector}
       </label>
-      {/* HACK: IE11 collapses td's height to 0, if it contains only an absolutely positioned label */}
```

Comment:
Can we split `supportsStickyPosition` and other changes in separate PRs to allow gradual rollout and rollback if anything unexpected happens?

Reply:
sure